.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    concurrency: release
+    permissions:
+      id-token: write
+      contents: write
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Python Semantic Release
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.11
```

Comment:
Doesn't it work with Python 3.12?

Reply:
Yes, it works. we usually use Python3.11 so i set that version, but i can change it

Reply:
Please proceed